src/api/person.js

```diff
@@ -24,6 +25,32 @@ person.fetch = id => {
   });
 };
 
+person.fetchAll = () =>
+  new Promise(resolve =>
+    person.fetchPage(resolve, 1));
+
+person.fetchPage = (resolve, pageNumber) => {
+  const pageToGet = `${personBaseUrl}?page=${pageNumber}`;
+  Vue.http.get(pageToGet).then(response => {
+    for (const result of response.data.results) {
```

Comment:
Could use an object with a constructor to encapsulate this logic and move it to a testable unit.